sky/spot/spot_state.py

```diff
@@ -83,7 +84,10 @@ def terminal_statuses(cls) -> List['SpotStatus']:
 
     @classmethod
     def failure_statuses(cls) -> List['SpotStatus']:
-        return [cls.FAILED, cls.FAILED_NO_RESOURCE, cls.FAILED_CONTROLLER]
+        return [
```

Comment:
Update L39 to say it includes setup time?

Reply:
Done. Thanks!